cypress/e2e/visual.regression.cy.js

```diff
@@ -0,0 +1,16 @@
+describe('visual regression tests', () => {
+
+    it('should display the Chart.js charts correctly', () => {
+        cy.visit('./examples/chartjs.html').wait(3000).compareSnapshot('chart.js')
+    });
+
+    it('should display the C3.js charts correctly', () => {
+        cy.visit('./examples/c3js.html').wait(3000).compareSnapshot('c3.js')
```

Comment:
For c3.js engine for generating charts, we can compare each generated chart with a golden master file. After that, we can use these golden master files for a documentation purpose.